src/_posts/2024-04-12-open-source-development.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: blog_post
+title: "Driving Progress: The Crucial Role of Open Source in Public Transportation Technology"
+subtitle:
+description: "Learn from the Compiler Team about the benefits of open source development"
+author: Milo Green
+excerpt: "Learn from the Compiler Team about the benefits of open source"
+date: 2024-04-12 00:00:00 +0200
+categories: compiler, engineering
+---
+
+<figure>
+    <img src="/assets/blog/2024/2024-4-mobility-illustration.jpg" alt="An illustration of smart transit where a smart bus is on a road with other cars and people on the sidewalk" />
+    <figcaption>An illustration of smart transit.</figcaption>
+</figure>
+
+In the era of rapid urbanization and the climate crisis, public transportation systems have become vital for ensuring sustainable mobility and reducing the impact of congestion and pollution. One unsung hero behind the scenes of modern public transit is open-source technology. When a technology is open source, it simply means that software has been produced in such a way that anyone can inspect, modify, and enhance it. This is important because it represents open collaboration, which many advocates believe helps facilitate greater public trust in the software while improving functionality and sustainability. Both [federal](https://www.cio.gov/2016/08/11/peoples-code.html) and state agencies, such as the [CA Department of Technology](https://cdt.ca.gov/wp-content/uploads/2018/05/TL-18-02-OSCodeReuse_2018-0419.pdf) have adopted strong open source first mandates for software solutions developed or acquired using public funds.
+
+Our goal at Compiler is simple - to help government agencies meet their strategic goals and better serve communities. We do that by providing customized solutions and impactful consulting services to improve IT workflows within organizations. One key component of this is working on code that is open source. We feel strongly that software and code developed using tax dollars should be produced as open source wherever possible, as this increases transparency, reduces opportunities for vendor lock-in, and delivers better and more cost-effective results to the public.
+
+<figure>
+    <img src="/assets/blog/2024/2024-4-metro-k-line.png" alt="A photograph of the Metro K line light rail train on the track in Los Angeles" />
+    <figcaption>Metro K line in Los Angeles</figcaption>
+</figure>
+
+Some of the benefits of open-source software development include:
+
```

Comment:
There's an oddly large amount of space between the line here and the first bullet point, I'm not sure if there's any way to fix that?

Reply:
This is a problem with the styling, nothing that can be fixed in the content itself.